pkg/ui/app/util/format.ts

```diff
@@ -1,23 +1,34 @@
 const kibi = 1024;
-const units: string[] = ["KiB", "MiB", "GiB", "TiB", "PiB", "EiB", "ZiB", "YiB"];
+const byteUnits: string[] = ["B", "KiB", "MiB", "GiB", "TiB", "PiB", "EiB", "ZiB", "YiB"];
+const durationUnits: string[] = ["ns", "µs", "ms", "s"];
 
 interface UnitValue {
   value: number;
   units: string;
 }
 
-export function BytesToUnitValue(bytes: number): UnitValue {
-  if (Math.abs(bytes) < kibi) {
-    return {value: bytes, units: "B"};
+// computePrefixExponent is used to compute an appopriate display unit for a value
+// for which the units have metric-style prefixes available. For example, the
+// value may be expressed in bytes, but we may want to display it on the graph
+// as a larger prefix unit (such as "kilobytes" or "gigabytes") in order to make
+// the numbers more readable.
+export function ComputePrefixExponent(value: number, prefixMultiple: number, prefixList: string[]) {
+  // Compute the metric prefix that will be used to label the axis.
+  let maxUnits = Math.abs(value);
+  let prefixScale: number;
+  for (prefixScale = 0;
```

Comment:
Not sure if it's necessarily more readable, but this could probably also be expressed as
```prefixScale = Math.log(maxUnits)/Math.log(prefixMultiple)```